schema/schema.py

```diff
@@ -132,4 +178,4 @@ def write_schema():
 
 
 if __name__ == "__main__":
-    generate_schema()
+    write_schema()
```

Comment:
Could we make this more modular? That way in the future when tests are added/considered, we could test each component a little more easily.